src/protocols/secure_channel/PairingSession.h

```diff
@@ -174,6 +174,29 @@ class DLL_EXPORT PairingSession : public SessionDelegate
         else
         {
             err = OnFailureStatusReport(report.GetGeneralCode(), report.GetProtocolCode());
+
+            if (report.GetGeneralCode() == Protocols::SecureChannel::GeneralStatusCode::kBusy &&
+                report.GetProtocolCode() == Protocols::SecureChannel::kProtocolCodeBusy)
+            {
+                if (!report.GetProtocolData().IsNull())
+                {
+                    Encoding::LittleEndian::Reader reader(report.GetProtocolData()->Start(),
+                                                          report.GetProtocolData()->DataLength());
+
+                    // TODO: CASE: Notify minimum wait time to clients on receiving busy status report #28290
+                    uint16_t minimumWaitTime = 0;
+                    err                      = reader.Read16(&minimumWaitTime).StatusCode();
```

Comment:
For a follow-up CL: make this boilerplate parsing get done in StatusReport as well.